### ISSUE TITLE
(Question) Would this work with Riiconnect24

### ISSUE THREAD

**Issue by Meganium97** [SwareJonge/Dolphin-Lua-Core#12]
Just wondering if it would work with this guide: https://wii.guide/riiconnect24-dolphin

**Comment by ghost** [comment]
yes, riiconnect24 works with any dolphin version (i think, i havent tested it in a while now)
